tests/phpunit/wp-tests-config.php

```diff
@@ -23,15 +23,33 @@
  * DO NOT use a production database or one that is shared with something else.
  */
 
+/**
+ * Returns an environment variable value if it exists, otherwise the default value.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $name Environment variable name.
+ * @param mixed $default A default value to use if the env variable is not set.
+ * @return mixed The environment variable value if it exsits, otherwise the default value.
+ */
+function get_env( $name, $default ) {
```

Comment:
Since this is only intended to be used here, how about we define this as a closure instead? Alternatively, we could add a namespace to the file. What do you think?

Reply:
> ...how about we define this as a closure instead?

@aaemnnosttv do you want to hide it from the global scope? Something like this?

```php
$get_env = function( $name, $default ) {
    ...
};

define( 'DB_USER', $get_env( 'WORDPRESS_DB_USER', 'root' ) );
```

Reply:
Yes that's what I meant 🙂 Either defining it as more of a temporary local function like this or namespacing the file to avoid adding a global function, but this looks fine to me 👍

Reply:
👍 Updated.